citools/console/commands/config.py

```diff
@@ -0,0 +1,22 @@
+import os
+import subprocess
+
+from cleo import Command
+from mozci.configuration import config
+
+
+class ConfigCommand(Command):
+    """
+    open mozci configuration file in editor
```

Comment:
Please change this to `Open your mozci configuration file in an editor.` for consistency.